packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -65,30 +68,37 @@ export class Runner {
 
         await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
 
+        let scanSucceeded: boolean;
         try {
             await this.webDriverTask.launch();
-            await this.scan(pageScanResult, scanMetadata.url);
-            this.logger.trackEvent('ScanRequestCompleted', undefined, { completedScanRequests: 1 });
+            scanSucceeded = await this.scan(pageScanResult, scanMetadata.url);
         } catch (error) {
+            scanSucceeded = false;
             const errorMessage = this.getErrorMessage(error);
             pageScanResult.run = this.createRunResult('failed', errorMessage);
             this.logger.logInfo(`Page scan run failed.`, { error: errorMessage });
-            this.logger.trackEvent('ScanRequestFailed', undefined, { failedScanRequests: 1 });
-            this.logger.trackEvent('ScanTaskFailed');
         } finally {
             const scanCompletedTimestamp: number = Date.now();
             const telemetryMeasurements: ScanTaskCompletedMeasurements = {
                 scanExecutionTime: (scanCompletedTimestamp - scanStartedTimestamp) / 1000,
                 scanTotalTime: (scanCompletedTimestamp - scanSubmittedTimestamp) / 1000,
+                completedScanTasks: 1,
             };
             this.logger.trackEvent('ScanTaskCompleted', undefined, telemetryMeasurements);
+            this.logger.trackEvent('ScanRequestCompleted', undefined, { completedScanRequests: 1 });
```

Comment:
I'm wondering if "completedScanRequests" value is always 1, what value does it add if we already know that it is a "ScanRequestCompleted" event?